pom.xml

```diff
@@ -9,11 +9,11 @@
   </parent>
 
 	<groupId>org.docx4j</groupId>
-	<artifactId>docx4j-parent</artifactId>	
+	<artifactId>docx4j-parent</artifactId>
```

Comment:
minor/question: usually i appreciate removing whitespace, but does it make more sense to ignore in this kind of situation where we are forking, just so as little as possible is actually changed?

Reply:
kind of yeah, at least from a "do as little work as necessary to get it functional" perspective. i'll try to refrain from too much white-space changes but sometimes I can't help myself